lib/styles/styles.dart

```diff
@@ -8,15 +8,16 @@ class AppStyle {
   late final _Text text = _Text();
   late final AppColors colors = AppColors();
   late final _Padding padding = _Padding();
+  late final _Spacers spacers = _Spacers()..padding = padding;
   late final _SeasonSizes seasonSizes = _SeasonSizes();
   late final _ContainerStyles containerStyles = _ContainerStyles();
   late final _BorderStyle borderStyle = _BorderStyle();
 }
 
 @immutable
 class _Text {
-  final TextStyle font = GoogleFonts.lexend();
-  final String? fontFamily = GoogleFonts.lexend().fontFamily;
+  final TextStyle font = const TextStyle();
+  // final String? fontFamily = GoogleFonts.lexend().fontFamily;
```

Comment:
This was throwing, so I took it out to work on the rest.

Reply:
Google Fonts probably required adding allowing network access on macOS! I just opened a PR to update the setup instructions in the readme. https://github.com/leighajarett/veggie_seasons_v2/pull/4